bin/list_bucket_metrics.js

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
wut ? What's a shebang doing in a js file ? (same for the following shell command, btw)
If it's documentation to tell how to run the file, I'd rather see it as a proper comment explaining things clearly.
Also, `#!/usr/bin/env node` should be fine...